docs/cluster-guide.md

```diff
@@ -0,0 +1,84 @@
+# Guide to setting up a 3-node cluster
+
+Welcome to the quick start guide for setting up a 3-node cluster using EventStoreDB. This guide covers how to prepare the environment, generate the required certificates, and how to configure and run the cluster.
+
+## Preparing the environment
+
+1. Create a folder named 'Cluster' on the desktop. 
+    - For production use, place the 'Cluster' folder in the root directory (C:).
+2. In 'Cluster', create subfolders: 'Node1', 'Node2', 'Node3'.
+3. Within each node, create these subfolders:
+    - 'Certificates' for storing the certificate and private key. 
+    - 'Data' for all data files.
+    - 'Index' for node indexes.
+    - 'Logs' for log files.
+4. Add a configuration file (.yml or .txt) in each node folder. For example:
+
+![Node folder example](./images/node-folder-example.png "Node folder example")
+
+## Generating the certificates
+
+1. Create a 'Generate_Certificate' folder on the desktop. 
+2. Download the latest version of the certificate generator from [EventStore es-gencert-cli-releases](https://github.com/EventStore/es-gencert-cli/releases) and unzip it into 'Generate_Certificate'.
+3. In the command line, navigate to: 
+
+`C:\Path\To\Folder\Generate_Certificate\es-gencert-cli_[Version]_Windows-x86_64`
+
+- Run the command to generate the root certificate and root private key: 
+`.\es-gencert-cli create-ca -out [Your Generate_Certificate Path]\ca`
+`C:\Path\To\Folder\Generate_Certificate\es-gencert-cli_[Version]_Windows-x86_64\ca`
+
+4. In the 'Generate_Certificate' directory, run the following commands one at a time for each node (change the path and node number as needed) to generate the certificate and private keys for each node.
+
+For example, if the certificate generator version is 1.2.1 and we’re generating the certificate and private keys for 'Node1':
+
+`.\es-gencert-cli create-node -ca-certificate C:\Path\To\Folder\Generate_Certificate\es-gencert-cli_1.2.1_Windows-x86_64\ca\ca.crt -ca-key`
+ `C:\Path\To\Folder\Generate_Certificate\es-gencert-cli_1.2.1_Windows-x86_64\ca\ca.key -out` 
+ `C:\Path\To\Folder\Cluster\Node1\certificates  -ip-addresses 127.0.0.1`
```

Comment:
@rmalik-es : little known trick if you're going the ip address route: 

you can use the full range of 127.x.x.x as address : 
* node 1 intIP & extIP = 127.0.0.10 
* node 2 intIP & extIP = 127.0.0.20
* node 3 intIP & extIP = 127.0.0.30

this allows to keep the standard ports (2113.1113)
and also is closer to what it will look like in a production scenarion